### PR TITLE
Fixing two typos in the "History" section of README

### PR DESCRIPTION
**Pull request by avian2**
Hi Philippe,

this change fixes two small errors: there was a mistake in the label above the *tablix.org/...* URL and there was an unnecessary "from" before the *code.google.com* URL.

Thanks continuing the development of this library and for nicely crediting the previous maintainers!

### CODE REVIEW

**Comment by pombredanne** [comment]
Thank you ++  and sorry for these mistakes!

**Comment by pombredanne** [comment]
@avian2 I pushed a new release with your fixes...
Thanks again!
Now since you announced you are discontinuing the development of `publicsuffix`, may be we could merge things together e.g. let go publicsuffix2 and rename it to publicsuffix including on PyPI? What do you think?

**Comment by avian2** [comment]
I'm not sure. I don't think it makes much sense to merge things now.

It's seems common in Python to have separate *foo*, *foo2*, etc. packages when major version is incremented (although as far as I can see, *publicsuffix2* is backward compatible with the last release of *publicsuffix*)

Looking at PyPi, it seems now both packages are being used in dependencies. *publicsuffix* lists [6 dependencies](https://libraries.io/pypi/publicsuffix/dependents), *publicsuffix2* lists [4 dependencies](https://libraries.io/pypi/publicsuffix2/dependents). I don't know how reliable these numbers are. I've manually checked a source of a few and *url* doesn't seem to actually depend on publicsuffix at all and *domain-connect* seems to have moved to *publicsuffixlist* (a different library that provides similar functionality). 

There's also "dependent repositories" count (not sure where they get that number from), that's currently 53 for *publicsuffix* and 16 for *publicsuffix2*.

[Debian package](https://tracker.debian.org/pkg/python-publicsuffix) seems to be marked for removal as part of the Python2 deprecation effort. The only dependency in Debian is [bley](https://tracker.debian.org/pkg/bley).

Searching for "publicsuffix" on PyPi, *publicsuffix* is first result, *publicsuffix2* is second. Since I've put "deprecated" at the very top of README and the link to *publicsuffix2* is the first in the list of alternatives, I think it's quite clear that *publicsuffix* shouldn't be used for new code.

So basically, removing any of the two packages on PyPi would break someone's code. Removing *publicsuffix* would break more things if PyPi numbers are to be trusted.
